### PR TITLE
Added DevOps, HTML&CSS and Linux resources to playground section 

### PR DESCRIPTION
**Pull request by Blesscode**
Added 1 new section name to the title name since DevOps title name was missing from the list in playground though 1 DevOps resource was already present below, Added 1 resource to the  DevOps section, added 2 Linux resources and  2 resources to HTML and CSS section for playground section
## What does this PR do?
Added ( DevOps, Linux, HTML & CSS) resources to the playground section

Add resource(s) | Remove resource(s) | Add info | Improve repo
Add resources

### Description
The playground learning section was missing sections such as DevOps Linux which was a little confusing to understand so to overcome that some game-based learning resources have been added moreover have added 2 project-based learning resources to the HTML and CSS section
The resources will work in such a way that whenever clicked on the link the resource will take the user to the resource page of the owner of the resource website we just added a link to the websites to this page so that it is easy for the user to find these amazing resources 

### Why is this valuable (or not)?
The playground learning section was missing sections such as DevOps Linux  which is  a little confusing to understand and overcome that some game-based learning resources have been added moreover 2 project-based learning resources are added to the HTML and CSS section

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: more/free-programming-interactive-tutorials-en.md [hunk 1]
```diff
@@ -30,6 +32,7 @@
 * [Language Agnostic](#language-agnostic)
     * [Operating Systems](#operating-systems)
 * [LaTeX](#latex)
+* [Linux](#linux)
```
Comment (eshellman) [inline more/free-programming-interactive-tutorials-en.md:35]: goes under operating systems.
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.

File: more/free-programming-interactive-tutorials-en.md [hunk 2]
```diff
@@ -117,6 +120,11 @@
 * [Smooth CoffeeScript, Interactive Edition](http://autotelicum.github.io/Smooth-CoffeeScript/interactive/interactive-coffeescript.html)
 
 
+### cmd
+
+* [Command Challenge](https://cmdchallenge.com) - Command Challenge
```
Comment (eshellman) [inline more/free-programming-interactive-tutorials-en.md:125]: cmd is not a thing.  this is linux/unix.
the resource fits better in the playgrounds list

File: more/free-programming-interactive-tutorials-en.md [hunk 3]
```diff
@@ -133,6 +141,12 @@
 * [Top 10 Data Science Projects for Beginners - KDnuggets](https://www.kdnuggets.com/2021/06/top-10-data-science-projects-beginners.html)
 
 
+### DevOps
```
Comment (eshellman) [inline more/free-programming-interactive-tutorials-en.md:144]: both of these fit better in playgrounds.

File: more/free-programming-interactive-tutorials-en.md [hunk 4]
```diff
@@ -352,6 +368,11 @@
 * [Learn the PHP Framework for Web Artisans](http://bootcamp.laravel.com) - Laravel Team
 
 
+### Linux
+
+* [OverTheWire](https://overthewire.org) - OverTheWire
```
Comment (eshellman) [inline more/free-programming-interactive-tutorials-en.md:373]: we have lots of crazy stuff like this in playgrounds

**Review by eshellman — CHANGES_REQUESTED** [review]
looks like these resources will be happier in playgrounds

**Comment by Blesscode** [comment]
> looks like these resources will be happier in playgrounds

Thank you for reviewing it and helping out, will make the changes accordingly

File: more/free-programming-interactive-tutorials-en.md (hunk 3 above)
Comment (Blesscode) [reply more/free-programming-interactive-tutorials-en.md]: Thanks for the review  and help i have moved them to the playground section

File: more/free-programming-interactive-tutorials-en.md (hunk 4 above)
Comment (Blesscode) [reply more/free-programming-interactive-tutorials-en.md]: Added these resources to the playgrounds 
Thank you for helping out

**Comment by eshellman** [comment]
no ALL CAPS (THE ODIIN PROJECT -> The Odin Project)
when you've made changes, please add `(changes made)` to the title of your PR

**Comment by Blesscode** [comment]
> Collaborator

Thank you so much for helping out 
I am doing it for the first time so I was a little confused with the tile 
Does (changes made) here refer to the overall changes that I have made or this specific name style change that I made or it's just I have to write the word "changes made" to the title 
For now, I am leaving the title with the changes I made overall 
Please suggest the changes that I should make to this

**Comment by eshellman** [comment]
I've reviewed about 200 PRs so far, and its not so easy to tell if requested changes have been made after review.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Comment by Blesscode** [comment]
> I've reviewed about 200 PRs so far, and its not so easy to tell if requested changes have been made after review.

Thank you so much for letting me know about this and helping me out with this PR .I hope I met all the requested criteria and changes correctly

**Comment by Blesscode** [comment]
> Thanks, and welcome to free-programming-books!

Thanks for having me

**Pull request merged**